apps/recovery-portal/src/main/webapp/password-recovery.jsp

```diff
@@ -264,6 +264,12 @@
         $(document).ready(function () {
 
             $("#recoverDetailsForm").submit(function (e) {
+
+                // Prevent clicking multiple times, and notify the user something
+                // is happening in the background.
+                const submitButton = $('#recoverySubmit');
```

Comment:
```suggestion
                const submitButton = $("#recoverySubmit");
```

Reply:
Fixed the linting issue in the commit a56cb5d. Thanks! :smile: